WindowsAppSDK-BuildVSIX.yml

```diff
@@ -0,0 +1,110 @@
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases for info on yaml ADO jobs
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+parameters:
+  # Version of the WindowsAppSDK used to build the Samples VSIX
+  - name: "WinAppSDKVersion"
+    displayName: "Windows App SDK version" 
+    type: string
+    default: '1.0.0-preview3'
```

Comment:
do we need to update this whenever WinAppSDKVersion in Directory.build.props is updated?

Reply:
I think the default probably shouldn't be set here. 

But the default value in Directory.build.props probably should be updated to the latest release?